files/fr/notes_08.json

```
@@ -1,4 +1,4 @@
 {
   "title": "Jour 8:",
-  "description": "Je pensais être tranquille, cela faisait 2 jours que je n’avais pas eu de nouvelles du chanteur, mais il m’a recontacté pour un autre de ses “job”. Au vu de ses phrases, je crois qu’il a lu mes notes précédentes, il faut que je trouve un moyen de les cacher, mon PC n’est plus assez sécurisé."
+  "description": "Je pensais être tranquille, cela faisait 2 jours que je n’avais pas eu de nouvelles du chanteur, mais il m’a recontactée pour un autre de ses “jobs”. À la vue de ses phrases, je crois qu’il a lu mes notes précédentes, il faut que je trouve un moyen de les cacher, mon PC n’est plus assez sécurisé."
```

"Au vu de" est une expression valide, je viens re-verifier, et je trouve que ca sonne mieux

```suggestion
  "description": "Je pensais être tranquille, cela faisait 2 jours que je n’avais pas eu de nouvelles du chanteur, mais il m’a recontactée pour un autre de ses “jobs”. Au vu de ses phrases, je crois qu’il a lu mes notes précédentes, il faut que je trouve un moyen de les cacher, mon PC n’est plus assez sécurisé."
```